contracts/factory/DAOFactory.sol

```diff
@@ -55,6 +53,8 @@ contract DAOFactory {
 
             acl.setPermissionManager(address(0), dao, appManagerRole);
             acl.setPermissionManager(_root, acl, permRole);
+        } else {
+            dao.initialize(baseACL, _root);
```

Comment:
I like it! Given the difference between the size of the code blocks, I would swap them: 
```solidity
if (address(regFactory) == address(0)) {
  dao.initialize(baseACL, _root);
} else {
  ...
}
```